src/main/scala/is/hail/methods/CalculateConcordance.scala

```diff
@@ -73,8 +66,8 @@ object CalculateConcordance {
 
     if (left.vSignature != right.vSignature)
       fatal(s"""Cannot compute concordance for datasets with different reference genomes:
-              |  left: ${left.vSignature.toPrettyString(compact = true)}
-              |  right: ${right.vSignature.toPrettyString(compact = true)}""")
+              |  left: ${ left.vSignature.toPrettyString(compact = true) }
+              |  right: ${ right.vSignature.toPrettyString(compact = true) }""")
```

Comment:
The current behavior for duplicate samples is all duplicates on the left are kept, and each gets it's own combiner. For duplicates on the right, one is randomly dropped by this line `rightIds.zipWithIndex.toMap`. I think this is fine, but we should add to the documentation. I can add on my concordance docs branch.

Reply:
Do you think it is cleaner to just require unique sample IDs?